models/cars/rookie.csv

```diff
@@ -1,36 +1,33 @@
-name,speed,acc,weight,multiplier,folder_name,stock
-RC Bandit,51.7,2.09,1.2,0.7,rc,TRUE
-Dr. Grudge,51.3,2.54,1.4,0.7,beatall,TRUE
-Sprinter XL,50.7,3.2,0.9,0.7,tc6,TRUE
-E-Razr,47.2,3.1,1.1,0.7,erazer,FALSE
-Hurricane,48.8,2.3,1.5,0.7,hurri,FALSE
-Iso-77A,48.8,3.03,1.33,0.7,iso77,FALSE
-Kanberra Kruiser,50.8,2.38,1,0.7,kkruise,FALSE
-Harvester,47.9,2.32,1.7,0.8,mud,TRUE
-Albatross GT,48.5,2.36,1.4,0.8,albatrossgt,FALSE
-Bed Bug,52.1,1.98,1.2,0.8,bedbug2,FALSE
-Condor GRV,51.5,2.73,1.5,0.8,condorgrv,FALSE
-El Gekko,51.3,2.01,1.2,0.8,gekko,FALSE
-Funk Flea,51.0,2.77,1.14,0.8,flea,FALSE
-High-Rod,49,2.17,1.9,0.8,rod,FALSE
-Hot Spot,51.3,2.36,1,0.8,hs,FALSE
-LR 64,54.1,1.55,1.7,0.8,fd64lost,FALSE
-Panorama,50.6,2.3,1.35,0.8,fd50pano,FALSE
-Road Star,50.3,2.49,0.8,0.8,roadstar,FALSE
-Col. Moss,52.8,2.17,1.5,0.9,moss,TRUE
-Volken Turbo,49.5,2.24,1.6,0.9,volken,TRUE
-BigVolt,52.8,2.36,2.3,0.9,bigvolt,TRUE
-Ferrada,54.1,2.06,1.2,0.9,ferrada,FALSE
-Nesbitt,53.5,1.81,1.5,0.9,nesbitt,FALSE
-Rebound 4X4,53.8,2.25,1.5,0.9,rebound,FALSE
-Gunior,51,2.67,1.3,1.0,fd114gunior,FALSE
-Toukka 4x4,49.1,2.47,2.2,1.0,toukka,FALSE
-Get Air,51.7,3.31,2.7,1.1,getair,FALSE
-Rouge,52,2.57,1.2,1.1,fd25sporty,FALSE
-Super Wheat,52.4,1.86,2.4,1.1,swheat,FALSE
-Dust Mite,50.5,1.98,1.2,1.2,mite,TRUE
-Piccino,50.1,3.11,0.85,1.2,midgetii,FALSE
-HSF-1,48.4,2.29,1.42,1.4,hsf1,FALSE
-Phat Slug,50.8,1.41,2.5,1.7,phat,TRUE
-Shiawase,41.6,1,5,2.0,shiawasecrimsonsolace,FALSE
-Creeki Leeki,49.9,2.16,1.2,3.0,gtcnd_creekileeki,FALSE
+name,speed,acc,weight,multiplier,folder_name,main_author,stock
```

Comment:
Let's keep it singular, just `author`. Otherwise it implicitly feels like there're more authors for every creation.

```suggestion
name,speed,acc,weight,multiplier,folder_name,author,stock
```

Reply:
Well, that's usually the case, there are few mods done from scratch by a single author

Reply:
What about the ones which do not have more than 1 author?

Reply:
Personally I think it's better to imply every mod was done by more than one person than the opposite. For example, a car could've been done by a single author, but that car is a remodel of another car done by a second author, which indirectly helped to make the car from the beginning.

Reply:
Well, that's not the point I's trying to make here. I's just wondering how that is relevant to us or the site. We could instead add an ID row to construct a RVW URL on our end and just add a referencing button on the car pages.

For example, if we had https://www.revoltworld.net/dl/argon/ in the pack, the `rvw_id` column value would be `argon`for that car entry.

If your idea is giving as much credit to the authors, I feel like doing what I'm proposing would be right thing to do. I did not do it earlier becuse there is no way I can assume the string that will complete RVW's URL.

Reply:
Can we check this suggestion about `rvw_id` later then? Also this suggest was done over `rookie.csv` which I already reverted to push it on another PR, so I guess I should press "Resolve conversation" button for now.

Reply:
Yeah, let's review the RVW ID feature later. I'll create an issue for it

Reply:
I've been checking that, just to find out that there are two ways the website generates these IDs.
The old way was with just the foldername, the current way takes the name of the mod, lowercase it, separates the words with '-' and removes any other special character.
I just noticed the existance of the old method after testing IDs in excel done with the new method.

Reply:
I moved the conversation here: https://github.com/Re-Volt-America/RVA-Data/issues/15